.env.template

```diff
@@ -2,6 +2,7 @@ DB_USERNAME=
 DB_PASSWORD=
 GOVUK_NOTIFY_API_KEY=
 DOMAIN=http://localhost:3000
+CURRENT_COHORT=2021
```

Comment:
I think you might have just made more problems for yourself :D 

I was thinking more along the lines of adding a comment linking to a trello card saying "TODO: Figure out how to change cohort". I am not convinced that sticking it in environment makes it any easier. 

But, what I am certain of, is that if you want it in an environment variable, you will have to add it to a few more places. If you globally search for `GIAS_API_ROOT` you will see what I mean. Otherwise, I don't think it's going to work very well :/

Reply:
I noticed I'll need to add it to the env files. But yeh I get your point. I'll just add the todo.